### PR TITLE
update readme to take advantage of new-project

### PR DESCRIPTION
**Pull request by deads2k**
Updates to the sample app readme that add instructions for properly authorizing a user to see a project and make use of the new-project command.

/cc @smarterclayton @liggitt

### CODE REVIEW

**Comment by deads2k** [comment]
In response to https://github.com/openshift/origin/pull/999#issuecomment-75398492

File: examples/sample-app/README.md [hunk 1]
```diff
@@ -104,7 +104,21 @@ the present working directory is the same directory as this README.
         $ export CURL_CA_BUNDLE=`pwd`/openshift.local.certificates/admin/root.crt
         $ sudo chmod +r "$KUBECONFIG"
 
-4. Deploy a private docker registry within OpenShift with the certs necessary for access to master:
+4. Bind a user to the `view` role in the default namespace so you can observe progress in the web console
+
+        $ openshift ex policy add-user view anypassword:test-admin
+
+5. *Optional:* View the OpenShift web console in your browser by browsing to `https://<host>:8444`.  Login using user, `test-admin` and any password.
```
Comment (liggitt) [inline examples/sample-app/README.md]: "the user `test-admin`"

Comment (liggitt) [inline examples/sample-app/README.md]: nest this under the "Optional: View in web console" step?

Comment (deads2k) [inline examples/sample-app/README.md]: This step is labelled optional.  You can look at two times.  Once to see the docker registry deployed and once to see the rest of the test resources created.

Comment (liggitt) [inline examples/sample-app/README.md]: ok, nm

**Comment by liggitt** [comment]
Remove the project.json, and the reference to downloading it in container-setup.md?

**Comment by liggitt** [comment]
not sure how aggressive we're being with examples following best practices (e.g. "your script that used project.json will now break because that's not how we want you creating it"). @bparees, you have an opinion?

**Comment by bparees** [comment]
@liggitt yeah we should try to keep the readme following best practices since it's probably something a lot of people will end up monkey-see-monkey-doing.

**Comment by deads2k** [comment]
@liggitt updated for comments.

**Comment by liggitt** [comment]
LGTM

**Comment by deads2k** [comment]
[merge]

**Pull request merged**